base_user_role/wizard/create_from_user.py

```diff
@@ -0,0 +1,46 @@
+from odoo import models, fields
+
+
+class CreateFromUserWizard(models.TransientModel):
+    _name = "base_user_role.create_from_user_wizard"
+    _description = "Create role from user wizard"
+
+    name = fields.Char("Name")
```

Comment:
```suggestion
    name = fields.Char(required=True)
```

- since recent version, string is not guessed from the field, so it's unnecessary to set it, if it is not different.
- name is required, otherwise, the call of the function ``create_from_user`` will fail is name is not set.